components/ProfileView/NFTsToMint.tsx

```diff
@@ -78,7 +78,7 @@ export const NFTsToMint: React.FC = () => {
             <SimpleGrid gap={8} columns={{ base: 1, md: 2 }}>
               {nftsToMint.slice(0, 2).map(ns => (
                 <MintNFTTile
-                  QCURL={`${QUESTCHAINS_URL}/chain/${ns.questChain.chainId}/${ns.questChain.address}`}
+                  QCURL={`${QUESTCHAINS_URL}/${ns.questChain.chainId}/${ns.questChain.address}`}
```

Comment:
shouldn't this be the network name instead of chainId ?

Reply:
i missed that, thanks!